presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/optimization/function/OperatorTranslators.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.jdbc.optimization.function;
+
+import com.facebook.presto.expressions.translator.TranslatedExpression;
+import com.facebook.presto.plugin.jdbc.optimization.JdbcSql;
+import com.facebook.presto.spi.function.ScalarFunction;
+import com.facebook.presto.spi.function.ScalarOperator;
+import com.facebook.presto.spi.function.SqlType;
+import com.facebook.presto.spi.type.StandardTypes;
+
+import static com.facebook.presto.plugin.jdbc.optimization.function.JdbcTranslationUtil.forwardBindVariables;
+import static com.facebook.presto.plugin.jdbc.optimization.function.JdbcTranslationUtil.infixOperation;
+import static com.facebook.presto.spi.function.OperatorType.ADD;
+import static com.facebook.presto.spi.function.OperatorType.SUBTRACT;
+
+public class OperatorTranslators
+{
+    private OperatorTranslators()
+    {
+    }
+
+    @ScalarOperator(ADD)
+    @SqlType(StandardTypes.BIGINT)
+    public static JdbcSql add(@SqlType(StandardTypes.BIGINT) TranslatedExpression<JdbcSql> left, @SqlType(StandardTypes.BIGINT) TranslatedExpression<JdbcSql> right)
```

Comment:
Nice yeah I noticed this as well this morning since this was resulting in a lot of multi-level isPresent checks throughout the call-stack making this quite dirty.